docs/storage/s3/parquet.md

```diff
@@ -0,0 +1,38 @@
+# Apache Parquet
```

Comment:
I don't see evaluation of minio and Parquet usage in the context of the providers. Please add

Reply:
Hmm, Minio doesn't have anything to do with that, as it would only download/upload a file. BTW Minio supports S3 select.